// Error Handling using try and catch

// let person1 = {
    //fname : 'love',
    //lname: 'Babber',
 // }

 // get fullName() {
    // return '$(person.fname)
  // }

  // // to call
  // console.log(erson1.fullName); // getter
  //person1.fulName = 'Niraj kumar Tiwari'; //setter
  // console.log(person1.fullNmae);


  // try catch to throw error
  // try {
    // person1.fullName = true;
    //}
    // catch (e){
        // alert(e)
    // }
  